### DISCUSSION TITLE
error updating from 7.9 to 7.21

### DISCUSSION THREAD

**Discussion by APerricone** [dlemstra/Magick.NET#737] [Help] [answered]
I am updating a project of mine from the version 7.9 to 7.21. And this line `MagickImage image = New MagickImage((System.Drawing.Bitmap)frame);` does not compile anymore.
I see the code is in IMagickImageExtentions but I am not able to use it...

**Comment by dlemstra** [comment, marked as answer]
You will need this new package: https://www.nuget.org/packages/Magick.NET.SystemDrawing/

**Reply by APerricone** [reply]
Many thanks
